src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -50,10 +50,11 @@ JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Control::ScrollbarState)
 
 JSON_FLAG_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::BellStyle)
 {
-    static constexpr std::array<pair_type, 4> mappings = {
+    static constexpr std::array<pair_type, 5> mappings = {
         pair_type{ "none", AllClear },
         pair_type{ "audible", ValueType::Audible },
-        pair_type{ "visual", ValueType::Visual },

```

Comment:
If you want to fake deprecate "visual", you'll probably want to keep this.

Reply:
Yep! Though it now maps to `Window | Taskbar`